src/app/components/recipe/recipe.component.spec.ts

```diff
@@ -7,6 +8,14 @@ import { MdCardModule, MdButtonModule, MdChipsModule, MdIconModule } from '@angu
 import { RecipeComponent } from './recipe.component';
 import { CardComponent } from './../card/card.component';
 
+const RECIPE_DATA = {
+    id: '1',
+    attributes: {
+        title: 'Recipe title',
+    }
+};
+const recipe: Recipe = new Recipe(null, RECIPE_DATA);
```

Comment:
Interesting that this is failing on travis but working for me locally. 

(https://github.com/ghidoz/angular2-jsonapi/issues/79)